crates/sdk/src/lib.rs

```diff
@@ -85,14 +86,20 @@ impl ProverClient {
                 }
```

Comment:
I think we should just get rid of the new function entirely. The only way its useable IMO is if you read the code, which is not ideal

Reply:
cc @yuwen01

Reply:
I've discussed that with Chris, the thing is `ProverClient::new()` is used in the default SP1 template, so removing it break a lot of people.